starter-code/javascript/main.js

```diff
@@ -7,54 +8,78 @@ var secDec      = document.getElementById('secDec');
 var secUni      = document.getElementById('secUni');
 var milDec      = document.getElementById('milDec');
 var milUni      = document.getElementById('milUni');
+var splits      = document.getElementById('splits');
 
+function showTime() {   // display time on the chronometer
+    setInterval(printTime, 100)
+}
```

Comment:
So I know we looked at this together, but I must say while is one solution, it is really not the most efficient way for doing this, because this `setInterval` here is being cleared anywhere and also it means you have two concurrent processes running.  You have access to `document` in your `chronometer.js` file as well.  Or you could pass the `printTime` function to your chronometer `startClick` function:
```
if (btnLeft.innerHTML == "START") {   // buttons on load: START and RESET
    chrono.startClick(printTime);
```
then in chronometer.js:
```
 Chronometer.prototype.startClick = function (printTime) {  // Start Click
    var that = this
    this.intervalId = setInterval(function() {
        that.currentTime += 1;
        console.log(that.currentTime)
        that.setTime();
        printTime()
     }, 1000)

};
```